docs/getting-started/index.md

```diff
@@ -0,0 +1,534 @@
+| testutil/      | Contains helper functions for testing                                                                                                                                          |
+| vue/           | Contains a Vue 3 web app template                                                                                                                                              |
+| x/             | Contains custom modules                                                                                                                                                        |
+| config.yml     | A configuration file for customising a chain in development                                                                                                                    |
+
+
+## Hello, Starport!
+
+To get started, let's get our blockchain up and running locally on a single node.
+
+### Starting a Blockchain
+
+You actually have a fully-functional blockchain already. To start it on your development machine, run the following command in the `blog` directory
+
+```
+starport serve
+```
+
+This will download dependencies, compile the source code into a binary called `blogd` (by default Starport uses the name of the repo + `d`), use this binary to initialize a single validator node, and start the node.
+
+![./images/api.png](./images/api.png)
+
+A validator node exposes two endpoints: [http://localhost:26657](http://localhost:26657) for the low-level Tendermint API and [http://localhost:1317](http://localhost:1317) for the high-level blockchain API.
+
+When you want to stop your blockchain, press Ctrl+C in the terminal window where it's running. In the development environment, Starport doesn't require you to restart the blockchain; changes you make in files will be automatically picked up by Starport.
+
+### Say "Hello, Starport"
+
+To get your Cosmos SDK blockchain to say "Hello", you will need to modify a protocol buffer file, create a keeper query function that returns data, and register a query function. Protocol buffer files contain proto `rpc`s that define Cosmos SDK queries and message handlers and proto `message`s that define Cosmos SDK types. `rpc`s are also responsible for exposing an HTTP API. [Keeper](https://docs.cosmos.network/v0.42/building-modules/keeper.html) is an abstraction for modifying the state of the blockchain and keeper functions let you query or write to the state. Registering a query needs to happen only once after you add the first query to your chain.
+
+In terms of workflow, developers usually work with proto files first to define Cosmos SDK messages, queries, message handlers and then implement the logic of these queries and message handlers in keeper functions.
+
+Let's start by creating a `posts` query:
+
+```
+starport query posts --response body,title
```

Comment:
let's do `starport query posts --response title,body` as in the rest of the tutorial, this is the referred order